packages/html/stories/PerimeterVariousImplementations.stories.ts

```diff
@@ -0,0 +1,253 @@
+/*
+Copyright 2024-present The maxGraph project Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { Graph, InternalEvent, RubberBandHandler } from '@maxgraph/core';
+import {
+  globalTypes,
+  globalValues,
+  rubberBandTypes,
+  rubberBandValues,
+} from './shared/args.js';
+// style required by RubberBand
+import '@maxgraph/core/css/common.css';
+
+export default {
+  title: 'Labels/PerimeterVariousImplementations',
+  argTypes: {
+    ...globalTypes,
+    ...rubberBandTypes,
+  },
+  args: {
+    ...globalValues,
+    ...rubberBandValues,
+  },
+};
+
+const Template = ({ label, ...args }: Record<string, any>) => {
+  const container = document.createElement('div');
+  container.style.position = 'relative';
+  container.style.overflow = 'hidden';
+  container.style.width = `${args.width}px`;
+  container.style.height = `${args.height}px`;
+  container.style.background = 'url(/images/grid.gif)';
+  container.style.cursor = 'default';
+
+  InternalEvent.disableContextMenu(container);
+
+  // Creates the graph inside the given container
+  const graph: Graph = new Graph(container);
+  // graph.setVertexLabelsMovable(true);
+  // graph.setConnectable(true);
+
+  // Enables rubberband selection
+  if (args.rubberBand) new RubberBandHandler(graph);
+
+  // Gets the default parent for inserting new cells. This is normally the first child of the root (i.e. layer 0).
+  const parent = graph.getDefaultParent();
+
+  // Adds cells to the model in a single step
+  graph.batchUpdate(() => {
+    // const rectangle1 = graph.insertVertex(parent, null, 'Rectangle 1', 20, 20, 80, 30);
+    const rectangle1 = graph.insertVertex({
+      parent,
+      value: 'Rectangle 1',
+      position: [20, 20],
+      size: [80, 30],
+    });
+    const rectangle2 = graph.insertVertex({
+      parent,
+      value: 'Rectangle 2',
+      position: [100, 100],
+      size: [80, 30],
+    });
+    graph.insertEdge({ parent, source: rectangle1, target: rectangle2 });
+    const rectangle3 = graph.insertVertex({
+      parent,
+      value: 'Rectangle 3\nnone perimeter',
+      position: [250, 20],
+      size: [80, 60],
+      style: {
+        perimeter: 'none',
```

Comment:
**note:** same effect by passing null, so we may use null instead and update the CellState.perimeter JSDoc to clarify this (and also when passing a string that doesn't relate to a registered perimeter)